### ISSUE TITLE
Content is not displaying

### ISSUE THREAD

**Issue by s-fog** [mostafaznv/nova-ckeditor#7]
Hi. Have very annoying issue)

Dependencies max updated. 

Field inside nova resource
`CkEditor::make('Информация', 'information')`

Have information field 
![image](https://user-images.githubusercontent.com/10043516/169004290-cc029ebc-1a73-4666-9526-36d64ecb53b7.png)

Content stored in database like that 
`<p>hfghfghfghfghfg</p>`

And it do not displaying, like screenshot. And i cant update this field as well. There is no accessors on this field. If i switch to reqular Textarea field, everything works as expected. So, it is Ckeditor issue or something

Can u check what is it? Thanks!

In nova 3, everything works perfectly.

----------------------------------------------------------------------------------------

Here is your component

```
<textarea ref="editor" class="hidden" :id="field.attribute" :class="errorClasses" :value="value" />
.....
methods: {
        setInitialValue() {
            console.log(this.field.value, 22)
            this.value = this.field.value || ''
        },
```

console log is `<p>hfghfghfghfghfg</p> 22`

Why it is not there, have no clue) Waiting for your help

**Comment by s-fog** [comment]
Here is your component

```
<textarea ref="editor" class="hidden" :id="field.attribute" :class="errorClasses" :value="value" />
.....
methods: {
        setInitialValue() {
            console.log(this.field.value, 22)
            this.value = this.field.value || ''
        },
```

console log is `<p>hfghfghfghfghfg</p> 22`

Why it is not there, have no clue) Waiting for your help

**Comment by mostafaznv** [comment]
Hi @s-fog 
Thanks for your report.

Bug fixed and published under 3.0.2
Please update the package and clear the browser's cache.

Feel free to reopen the issue or to submit a new one if you meet any problems.
